Automattic/jetpack
# Boost: Critical CSS and other small fixes

## Proposed changes:

* Move styles from global critical-css.scss file to their components (remove ones that aren't used);
* Update JS and CSS files to use path aliases;
* Remove unnecessary spacing below Critical CSS status section (when it's last).

<details><summary>Spacing before</summary>

![CleanShot 2024-01-22 at 12 57 32](https://github.com/Automattic/jetpack/assets/11799079/6f45813e-5ce0-4ce5-a002-f63d401fc457)

</details>

<details><summary>Spacing now</summary>

![CleanShot 2024-01-22 at 12 58 47](https://github.com/Automattic/jetpack/assets/11799079/265bd734-d530-4bb9-b432-9420e972f503)


</details>


### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/35010

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Make sure UI isn't broken.
